uniflow-android-test/src/main/java/io/uniflow/android/test/SimpleTestObservers.kt

```diff
@@ -38,9 +38,9 @@ class TestViewObserver {
     fun assertReceived(vararg any: UIData) = verifySequence(*any)
     fun verifySequence(vararg testingData: UIData) {
         val testingValues = testingData.toList()
-        assertEquals(values.size, testingValues.size, "Incorrect size of list ")
+        assertEquals(testingValues.size, values.size, "Incorrect size of list ")
         values.forEachIndexed { index, uiData ->
-            assertEquals(uiData, testingValues[index],
+            assertEquals(testingValues[index], uiData,
```

Comment:
it needs to be assertEquals(uiData, testingValues[index], where assertEquals(expected, actual)